Src/lib/UL/uart_manager/uart_manager.c

```diff
@@ -70,9 +73,72 @@ e_uart_manager_err_t send_to_uart(led_num_t led_num, led_ctrl_t led_state)
         return e_uart_manager_err_invalid_argument;
     }
 
-    if (uart_send(&uart, message) != e_uart_err_ok)
+    if (uart_send(&uart, send_message) != e_uart_err_ok)
     {
         return e_uart_manager_err_invalid_argument;
     }
     return e_uart_manager_err_ok;
 }
+
+e_uart_manager_err_t convert_receive_package(uint8_t *receive_message)
```

Comment:
You should send receive_message length as input arg also

Reply:
Done